admin/sql/create_tables.sql

```diff
@@ -1,11 +1,12 @@
 BEGIN;
 
 CREATE TABLE lowlevel (
-  id          SERIAL,
-  mbid        UUID NOT NULL,
-  build_sha1  TEXT NOT NULL,
-  lossless    BOOLEAN                  DEFAULT 'n',
-  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+  id              SERIAL,
+  gid             UUID    NOT NULL,
+  build_sha1      TEXT    NOT NULL,
+  lossless        BOOLEAN                  DEFAULT 'n',
+  submitted       TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
+  is_mbid         id_type NOT NULL
```

Comment:
Now that this is a type which indicates many things, I don't think `is_mbid` is a good name. In the ticket I suggested that we call it `gid_type` (http://tickets.musicbrainz.org/browse/AB-135?focusedCommentId=39881&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#action_39881)